daml-lf/language/src/main/scala/com/digitalasset/daml/lf/language/PackageInterface.scala

```diff
@@ -277,7 +277,7 @@ private[lf] class PackageInterface(signatures: PartialFunction[PackageId, Packag
     lookupInterface(ifaceName, context).flatMap(
       _.choices
         .get(chName)
-        .toRight(LookupError.NotFound(Reference.TemplateChoice(ifaceName, chName), context))
+        .toRight(LookupError.NotFound(Reference.InterfaceChoice(ifaceName, chName), context))
```

Comment:
tangential change, this used to give the wrong error message when the choice was not found

Reply:
Good catch.